### DISCUSSION TITLE
Mobile Functionality is required for Apache Superset

### DISCUSSION THREAD

**Discussion by radhakvnr** [apache/superset#18532] [Ideas]
**Is your feature request related to a problem? Please describe.**

Hi, we are using superset as BI tool in our company and we have the mobile device requirement. We would like to check the information and road map for the support of superset on mobile devices.

**Describe the solution you'd like**
mobile functionality is  inevitable.
Mobile being pervasive.  The transparent mobile support will significantly boost Superset adoption in the industry
Most vendor based BI tools  like MicroStrategy, Tableau absolutely supports the Mobile functionality and therefore the Superset does need that.

**Describe alternatives you've considered**
We would require Mobile device support for Apache Superset.

**Additional context**

**Comment by srinify** [comment]
Hi @radhakvnr you're correct that Superset doesn't currently work in a mobile friendly environment. While many people have interest in the outcome here, this is a large undertaking and we haven't seen much interest yet from the community to step up here and drive the implmentation.

Is this something you're interested in?

**Comment by himanshpal** [comment]
Having an even trimmed down mobile version of Superset would be super appreciated. Many folks with-in the community, including me can contribute to it if we plan to focus on it. Not having one requires either teams to build in-house or go with various third-party vendors.
